ci/build/build-docker-images.py

```diff
@@ -604,7 +598,7 @@ def push_and_tag_images(
     date_tag = datetime.datetime.now().strftime("%Y-%m-%d")
     sha_tag = _get_commit_sha()
     if _release_build():
-        release_name = re.search("[0-9]+\.[0-9]+\.[0-9].*", _get_branch()).group(0)
```

Comment:
this regex matching is also quite relaxed, like it will match with `00.00.0abcd!@#$% 
< s p a c e s >`

and is copied to release test infra.. so I want to kill it here..

Reply:
The new logic looks good (and is much simpler)! I think the old regex should have been something like[1] to handle `2.1.0.dev0` 'releases' in addition to normal `2.1.0` releases.

[1]